exercises/beer-song/BeerSongTest.swift

```diff
@@ -0,0 +1,24 @@
+#if swift(>=3.0)
+    import XCTest
+#endif
+
+class BeerSongTest: XCTestCase {
+
+    var beerSong: BeerSong!
+    override func setUp() {
+        super.setUp()
+        beerSong = BeerSong()
+    }
+
```

Comment:
Take a look at this other similar excessive. https://github.com/exercism/xswift/blob/master/exercises/twelve-days/TwelveDaysTest.swift

You don't have to do as many but I would say you would want at least 4 total.